src/main/java/me/kavin/piped/server/handlers/auth/AuthPlaylistHandlers.java

```diff
@@ -283,8 +283,7 @@ public static byte[] addToPlaylistResponse(String session, String playlistId, Li
         }
     }
 
-    public static byte[] removeFromPlaylistResponse(String session, String playlistId, int index) throws IOException {
-
+    public static byte[] removeFromPlaylistResponse(String session, String playlistId, Integer index) throws IOException {
```

Comment:
I don't like this approach, I would rather create a new method for clearing a playlist!

Reply:
Sure!